pkgs/os-specific/linux/cifs-utils/default.nix

```
@@ -14,6 +14,13 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ kerberos keyutils pam talloc ];
 
+  patches = [
+    (fetchpatch {
+      url = "https://attachments.samba.org/attachment.cgi?id=16148"; # CVE-2020-14342
```

```suggestion
      name = "CVE-2020-14342.patch";
      url = "https://attachments.samba.org/attachment.cgi?id=16148";
```

Setting the CVE identifier as the patch name enables tools like broken.sh to identify these patches.